addons/mail/static/src/components/discuss_mobile_mailbox_selection/discuss_mobile_mailbox_selection.xml

```diff
@@ -7,7 +7,7 @@
                 <button class="o_DiscussMobileMailboxSelection_button btn btn-secondary"
                     t-att-class="{
                         'active o-active': discussView.discuss.thread === mailbox,
-                    }" t-on-click="_onClick" t-att-data-mailbox-local-id="mailbox.localId" type="button"
+                    }" t-on-click="() => discussView.onClickMobileMailboxSelectionItem(mailbox)" t-att-data-mailbox-local-id="mailbox.localId" type="button"
```

Comment:
I believe that the data attribute can be removed now since `mailbox` is directly supplied in the arguments, and I don't think this attribute had any other purpose than retrieving `mailbox`.

Reply:
You're right; too bad 🙁